### DISCUSSION TITLE
preSave for all collections

### DISCUSSION THREAD

**Discussion by lorensansol** [StaticJsCMS/static-cms#970] [Q&A]
I am using the beta of Static CMS, and in the `Events Hooks` I have verified that to apply the `preSave` method it is necessary to indicate the `collection`.

But I want to apply it to all collections and not indicating the `collection` parameter does not work.

Any ideas?

**Comment by KaneFreeman** [comment]
I can see where there would be a use case for that. I've relaxed the requirements for registering events. If you don't specify a collection, file or field, your handler will fire for all events. This is available in `v4.0.0-beta.2`.

Updated docs for events: https://next.staticcms.org/docs/cms-events
